src/zoid/auth/props.js

```diff
@@ -19,10 +19,19 @@ export type OnCancelActions = {|
     
 |};
 
+export type OnErrorData = {|
```

Comment:
The `onError` prop is actually part of zoid: https://github.com/krakenjs/zoid/blob/68b90553425673a0fbc4cb17149c980d01efbdf0/src/component/props.js#L63. I don't think we need to define it here. I noticed that the checkout-components repo does not define it. It just relies on it as part of zoid in the test suite.